command/CommandController.js

```diff
@@ -228,6 +237,7 @@ async function processArticleRequest(req, summaryOnly, extendedData) {
   } else {
     result = { url: audioUrl };
   }
+  result.offset_ms = 0;
```

Comment:
It would be clearer if this 0 value was set in the /article route as an `else` condition if we don't find the astat (line 139 above). I guess it's being done here to ensure this base value is always present (i.e. for /summary requests too) -- there is value to that too. It's just a little confusing to have these two cases separated across functions. Maybe a comment here would be helpful, e.g. "Initally set offset to 0 and assume we'll start at the beginning of the file, then overwrite it later if necessary." ?

Reply:
Yes I put that there so that /summary also has this value. Adding a comment to explain